en/docs/assets/attachments/103334533/103334540.ps1

```
@@ -1,26 +1,26 @@
+    Foreach-Object { $_ -replace "org.wso2.carbon.apimgt.usage.publisher.APIMgtThrottleUsageHandler", "org.wso2.carbon.apimgt.gateway.handlers.analytics.APIMgtThrottleUsageHandler" } |
+    Foreach-Object { $_ -replace "org.wso2.carbon.apimgt.usage.publisher.APIMgtResponseHandler", "org.wso2.carbon.apimgt.gateway.handlers.analytics.APIMgtResponseHandler" } |
+    Foreach-Object { $_ -replace "org.wso2.carbon.apimgt.usage.publisher.APIMgtFaultHandler", "org.wso2.carbon.apimgt.gateway.handlers.analytics.APIMgtFaultHandler" } |
+    Foreach-Object { $_ -replace "org.wso2.carbon.apimgt.usage.publisher.APIMgtUsageHandler", "org.wso2.carbon.apimgt.gateway.handlers.analytics.APIMgtUsageHandler" } |
+    Foreach-Object { $_ -replace "org.wso2.carbon.apimgt.usage.publisher.APIMgtGoogleAnalyticsTrackingHandler", "org.wso2.carbon.apimgt.gateway.handlers.analytics.APIMgtGoogleAnalyticsTrackingHandler" } |
+    Foreach-Object { $_ -replace "org.wso2.carbon.mediator.cache.digest.DOMHASHGenerator", "org.wso2.carbon.mediator.cache.digest.REQUESTHASHGenerator" } |
+    Foreach-Object { $_ -replace "org.wso2.caching.digest.REQUESTHASHGenerator", "org.wso2.carbon.mediator.cache.digest.REQUESTHASHGenerator" } |
+    Foreach-Object { $_ -replace "<handler class=`"org.wso2.carbon.apimgt.gateway.handlers.security.APIAuthenticationHandler`"/>", "<handler class=`"org.wso2.carbon.apimgt.gateway.handlers.security.APIAuthenticationHandler`">`n`t`t<property name=`"RemoveOAuthHeadersFromOutMessage`" value=`"true`"/>`n`t</handler>" } |
+    Set-Content $file.PSPath
+}
+
+Write-Host "migration completed."
```

PSScriptAnalyzer (1.24.0)

[warning] File '103334540.ps1' uses Write-Host. Avoid using Write-Host because it might not work in all hosts, does not work when there is no host, and (prior to PS 5.0) cannot be suppressed, captured, or redirected. Instead, use Write-Output, Write-Verbose, or Write-Information.

(PSAvoidUsingWriteHost)